tests/lib/snaps/snapctl-hooks/meta/hooks/configure

```diff
@@ -67,7 +67,9 @@ test_get_nested() {
       exit 1
   fi
   expected_output='{\n\t"key1": "a",\n\t"key2": "b"\n}'
-  if ! ( echo "$output" | grep -z -q -P "$expected_output" ); then
+  # note: "echo" is a built-in of sh and doesn't support -e flag, use /bin/echo.
```

Comment:
This should work in spread fwiw, it uses "bash" as its shell.

Reply:
It didn't; it's a hook, so it runs with whatever core image has, I didn't investigate further.